packages/venia-ui/lib/queries/createAccount.graphql

```diff
@@ -0,0 +1,19 @@
+mutation createAccount(
+  $email: String!
+  $firstname: String!
+  $lastname: String!
+  $password: String!
+) {
+  createCustomer(
+    input: {
+      email: $email
+      firstname: $firstname
+      lastname: $lastname
+      password: $password
+    }
+  ) {
+    customer {
+      email
```

Comment:
I tried removing the `email` and `customer` props but you are required to request _something_ from the response. So even though the UI doesn't do anything with this data, we still have to ask for it. Not a big deal, IMO.

Reply:
Good to know, and good that we're requesting a very small piece of data. Mind dropping a comment in the file saying as much?

Reply:
Actually I don't think I should. This is just how GraphQL works: https://github.com/apollographql/graphql-tools/issues/277

Reply:
Yeah, the comment request was just to help folks learn ;)

No big deal

Reply:
Added!